public/css/filter.css

```
@@ -0,0 +1,127 @@
+.filter {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.search-bar {
+  margin: 10px;
+}
+
+.dropdown {
+  background-color: #eee;
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+  
+.dropdown ul {
+  background-color: #eee;
+  border: 1px solid #005a9c;
+  border-top-width: 5px;
+  border-radius: 0 0 4px 4px;
+  display: block;
+  list-style-type: none;
+  margin: 0;
+  min-width: 200px;
+  padding: 0;
+  position: absolute;
+  top: 100%;
+}
+  
+.dropdown li {
+  margin: 0;
+}
+  
+.dropdown>li {
+  display: flex;
+  position: relative;
+}
+  
+.dropdown ul a {
+  border: 0;
+  color: #000;
+  display: block;
+  margin: 0;
+  padding: 0.5em 1em;
+  text-decoration: underline;
+  font-size: 15px;
+  display: flex;
+  text-decoration: none; 
```

```suggestion
  text-decoration: none;
```
Et un espace ici en trop